bl-plugins/welcome/languages/en.json

```diff
@@ -1,7 +1,8 @@
 {
-	"plugin-data":
-	{
-		"name": "Welcome",
-		"description": "Shows a greeting message in the dashboard."
-	}
-}
\ No newline at end of file
+    "plugin-data":
+    {
+        "name": "Last pages",
```

Comment:
this files is incorrect, looks like is from the plugin "last pages'

Reply:
Sorry! It should be the right one now.

Reply:
Sorry! It should be the right one now.